code/controllers/subsystem/tickets/SSmentor_tickets.dm

```diff
@@ -71,6 +72,21 @@ GLOBAL_REAL(SSmentor_tickets, /datum/controller/subsystem/tickets/mentor_tickets
 	if(message_key == null)
 		T.staffAssigned = null //if they cancel we dont need to hold this ticket anymore
 		return
+	if(message_key == "NCT Dispatch")
+		var/nct_active = list()
+		for(var/mob/living/carbon/human/trainer as anything in GLOB.human_list) // Let's check if we have any active NCTs
+			if(trainer.mind?.assigned_role != "Nanotrasen Career Trainer")
+				continue
+			nct_active += trainer
+		if(!length(nct_active))
+			to_chat(usr,"There are no active NCTs. Autoresponse canceled.") // If we don't, don't solve the ticket and then send feedback.
+			return
+		var/mob/living/carbon/human/trainee = get_mob_by_ckey(T.client_ckey)
+		for(var/mob/living/carbon/human/nct as anything in nct_active)
+			if(!istype(nct.l_ear, /obj/item/radio/headset) && !istype(nct.r_ear, /obj/item/radio/headset)) // If the NCT doesn't have a headset, ignore it.
+				continue
+			to_chat(nct, "<span class='notice'>Incoming priority transmission from Nanotrasen Training Center.  Request information as follows: </span><span class='specialnotice'>Career Trainer, we've received a request from an employee. They're called [trainee.real_name], they're a [trainee.mind.assigned_role]. See if they need any help.</span>")
```

Comment:
Can you check to make sure this message doesn't get sorted into two different chat tabs? `notice` will be filtered to the Info tab, while `specialnotice` will probably end up in Unsorted.

Reply:
Okay just wanted to be sure. It probably just sorts the whole message based on the first class used. I didn't know if having two different in-line changes would fragment the message or not.